package.json

```diff
@@ -86,10 +88,16 @@
 	},
 	"ava": {
 		"files": [
-			"test/*.js"
+			"test/*.ts"
+		],
+		"babel": false,
+		"compileEnhancements": false,
+		"extensions": [
+			"js",
+			"ts"
 		],
 		"require": [
-			"esm"
```

Comment:
No need for esm as TypeScript covers that.

Reply:
Great! We can also drop the `esm` dependency then

Reply:
Yup, already dropped it. 🔥

Reply:
https://github.com/sindresorhus/refined-github/blob/2a71a833754d53e649a67318aaa48776df2e053b/package.json#L49

Still seeing it in the current commit

Reply:
Oops. Took it out of the AVA config not the actual package. My bad. 🙃